src/Parser/Warlock/Affliction/Modules/Features/AgonyUptime.js

```diff
@@ -9,31 +9,49 @@ import SpellIcon from 'common/SpellIcon';
 import { formatPercentage } from 'common/format';
 import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
 import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
 
 class AgonyUptime extends Analyzer {
   static dependencies = {
     enemies: Enemies,
     combatants: Combatants,
   };
 
-  suggestions(when) {
+  get suggestionThresholds() {
```

Comment:
I think making two suggestion thresholds, one for each version (with talent & without talent) would be more maintable. Also please don't include the text in the suggestion thresholds. I get why you did it here and I thought about it too, but my conclusion is that if we do want to do that eventually, it should include all props including icon, actual and recommended and by that time we're basically rewriting how suggestions are defined entirely and I wonder if there's any benefit to that. Long story short: don't put text in there unless you're willing to commit to all props.